_config.yml

```diff
@@ -28,5 +28,8 @@ expo:             http://expo.getbootstrap.com
 
 cdn:
   css:            https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css
+  css_hash:       sha384-pdapHxIh7EYuwy6K7iE41uXVxGCXY0sAjBzaElYGJUrzwodck3Lx6IE2lA0rFREo
```

Comment:
I used https://srihash.org , instructions are also on that page.

    cat FILENAME.js |
    openssl dgst -sha384 -binary |
    openssl enc -base64 -A

For porting to 4, it might be best to make this apart of building the documentation (if possible).
-- 
Chris Barry

On September 28, 2015 11:53:01 PM EDT, Chris Rebert <notifications@github.com> wrote:
>> @@ -28,5 +28,8 @@ expo:             http://expo.getbootstrap.com
>>  
>>  cdn:
>>    css:           
>https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css
>> +  css_hash:      
>sha384-pdapHxIh7EYuwy6K7iE41uXVxGCXY0sAjBzaElYGJUrzwodck3Lx6IE2lA0rFREo
>
>Should probably include a comment here saying how to generate the hash
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/twbs/bootstrap/pull/17729/files#r40632525